source/includes/k8s/deploy-operator.rst

```diff
@@ -74,22 +102,21 @@ The Kubernetes TLS API uses the CA signature algorithm for generating new TLS ce
 MinIO recommends ECDSA (e.g. `NIST P-256 curve <https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-4.pdf>`__) or EdDSA (e.g. :rfc:`Curve25519 <7748>`) TLS private keys/certificates due to their lower computation requirements compared to RSA.
 See :ref:`minio-TLS-supported-cipher-suites` for a complete list of supported TLS Cipher Suites.
 
-The Operator cannot complete initialization if the Kubernetes cluster is 
-not configured to respond to a generated CSR. Certain Kubernetes 
-providers do not specify these configuration values by default. 
+The Operator cannot complete initialization if the Kubernetes cluster is not configured to respond to a generated CSR. 
+Certain Kubernetes providers do not specify these configuration values by default. 
 
-To verify whether the ``kube-controller-manager`` has the required 
-settings, use the following command. Replace ``$CLUSTER-NAME`` with the name 
-of the Kubernetes cluster:
+To verify whether the ``kube-controller-manager`` has the required settings, use the following command:
```

Comment:
Would something like `To verify whether the kube-controller-manager is configured correctly` work instead of `has the required settings`? (I'm not sure how exacting these needed settings are.)

Reply:
The needed configurations are the cluster signing key and cert files. Rewording this sentence to specify those.